WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -879,6 +879,16 @@ - (BlogDetailsRow *)adminRow
     return row;
 }
 
+- (BlogDetailsRow *)applicationPasswordManagementRow
+{
+    __weak __typeof(self) weakSelf = self;
```

Comment:
I think it belongs to "Site Settings".

Reply:
Application passwords are bound to user, not site. On website, it's under user profile.

However, the Me tab (where we should user profile in the app) is hidden when only self-hosted site is added, which is probably an issue in itself. The app should allow user to go to user profile and app settings.

Reply:
Good point. Not sure where it should go, but probably not here.
"Me" tab is for the wp.com accounts. We should be showing your self-hosted site profile somewhere though.

Reply:
This one is tricky – for WP.org sites, it's tied to the site.

For WP.com, it's tied to the user.

And long-term, [we should support multiple WP.com accounts](https://github.com/wordpress-mobile/WordPress-iOS/issues/6015).

For WP.org sites, the application passwords _must_ be in this location, at least (but we could have a separate section under "Me" that lets you do the same thing)

Reply:
> This one is tricky – for WP.org sites, it's tied to the site.

The application passwords are tied to the user, aren't they?

<img width="320" alt="Screenshot 2024-08-20 at 9 37 12 AM" src="https://github.com/user-attachments/assets/4d693d30-1a86-42d1-a125-9d903aa226a2">
<img width="320" alt="Screenshot 2024-08-20 at 9 37 16 AM" src="https://github.com/user-attachments/assets/0cbe2ad3-1a12-46ce-b5d2-1f3784274b36">

I have only one user on this local site, but I imagine I can't see application passwords for other users.

The issue is that there is no profile management support for wp.org sites in the app.

Reply:
> However, the Me tab (where we should user profile in the app) is hidden when only self-hosted site is added, which is probably an issue in itself. The app should allow user to go to user profile and app settings.

Correction: There is a "Me" row in the homepage, where users can access App Settings and other things.

<img width="320" src="https://github.com/user-attachments/assets/7bf43cfd-01b2-4d99-bcec-33381667dc12"/>